examples/mlflow_artifacts/README.md

```diff
@@ -0,0 +1,54 @@
+This directory contains a set of files for demonstrating the MLflow Artifacts Service.
+
```

Comment:
I'll insert a new section here that briefly explains what the MLflow Artifacts Service does.

```suggestion
## What does the MLflow Artifacts Service do?
...
```